app/src/main/java/com/google/droidjump/leveldata/JSONKeys.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.droidjump.leveldata;
+
+/**
+ * Stores keys for level properties in JSON files.
+ */
+public interface JSONKeys {
```

Comment:
I am not sure whether these variables should be in the common class of constants or we should create a new class. 

Are there any benefits for having this new class?

Reply:
I did so because these constants are used only in the leveldata package, combined by the fact that they are json key-strings, and don't make sense for the rest of the app. But there is no particular benefits. So I can move them to GameConstants if you see fit.